bip-tap-proof-file.mediawiki

```diff
@@ -129,6 +129,16 @@ blob with the following format:
 * type: 10 (<code>challenge_witness</code>)
 ** value:
 *** [<code>...*byte</code>:<code>challenge_witness</code>]
+* type: 11 (<code>asset_genesis</code>)
+** value:
+*** [<code>32*byte</code>:<code>first_prev_out_hash</code>]
+*** [<code>u32</code>:<code>first_prev_out_index</code>]
+*** [<code>BigSize</code>:<code>tag_len</code>]
+*** [<code>tag_len*byte</code>:<code>tag</code>]
+*** [<code>BigSize</code>:<code>metadata_len</code>]
+*** [<code>metadata_len*byte</code>:<code>metadata</code>]
+*** [<code>u32</code>:<code>output_index</code>]
+*** [<code>u8</code>:<code>type</code>]
```

Comment:
Is this the asset type? Looks like we still repeat it below on the TLV serialization.

Reply:
Yes, this i the asset type. I think it is okay to repeat it here, since this is the proof file, and you need it to derive the asset_id